src/slic3r/GUI/Field.hpp

```diff
@@ -41,7 +41,120 @@ wxString double_to_string(double const value, const int max_precision = 4);
 wxString get_thumbnail_string(const Vec2d& value);
 wxString get_thumbnails_string(const std::vector<Vec2d>& values);
 
-class Field {
+class UndoValueUIManager
```

Comment:
Just trying to understand here: if I'm not wrong, this micro refactoring is not directly related to the EditGCodeDialog, right?

Reply:
According to the commits from PS, it was a refactoring done in preparation for the dialog. It is still fully compatible. The EditUIManager IS directly related.